public/js/paystack-forms-public.js

```diff
@@ -385,6 +385,7 @@ function KkdPffPaystackFee() {
                               var quantity = data.quantity;
                               // console.log(firstName+ " - "+lastName);
                               if (data.plan == "none" || data.plan == "" || data.plan == "no") {
+                                console.log(data)
```

Comment:
@lukman008  do we want to leave the console?

Reply:
@steveamaza I'll make a different PR